app/src/main/res/layout/fragment_add_login.xml

```
@@ -42,8 +42,6 @@
         app:layout_constraintEnd_toEndOf="@id/hostnameHeaderText"
         app:layout_constraintStart_toStartOf="@id/hostnameHeaderText"
         app:layout_constraintTop_toBottomOf="@id/hostnameHeaderText"
-        app:helperTextEnabled="true"
-        app:helperText="@string/add_login_hostname_invalid_text_3"
```

This seems like an important hint regarding what URLs are accepted. Why was it removed?

---

I moved it from being a hint to being an error, partially because previously the backend silently discarded otherwise-valid new logins that did not start with `http(s)`, and partially because showing the hint to the user when no hostname has been entered might make the user think that they still have to manually add the protocol themselves.

---

Can we still keep it the initial hint also?
It's a design that users are accustomed to and helps to more gently inform about the URL restrictions than through an immediate error.
(Also opened https://github.com/mozilla-mobile/fenix/issues/27839 to not be so quick on showing errors)

---

The problem I have with leaving the hint is that it sort of undermines the auto-addition of `https://`, at least with the current string.

What could be a good middle-ground is a new text hint that states something like "hostname must start with http(s)://. https:// will be added if no protocol is entered manually".

---

Adding a new string requires UX approval.
The proposed one would modify the UI a bit being quite long.